src/env.h

```diff
@@ -599,7 +599,9 @@ class Environment {
 
   enum Flags {
     kNoFlags = 0,
-    kIsMainThread = 1
+    kIsMainThread = 1 << 0,
+    kControlsProcess = 1 << 1,
```

Comment:
This is a bit of bike shedding..but why not use `Owns` since we also have `kOwnsInspector`? Also to me `process` is a bit ambiguous (could also mean the process object) so something like `kOwnsProcessStates` may be clearer

Reply:
@joyeecheung Done, changed to `kOwnsProcessState`/`owns_process_state()`/`ownsProcessState` :)